src/main/kotlin/com/example/noswimmingserver/domain/common_user/domain/repository/CustomUserRepositoryImpl.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.noswimmingserver.domain.common_user.domain.repository
```

Comment:
🚫 **[ktlint]** <package-name> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Package name must not contain underscore (cannot be auto-corrected)